src/presentation/components/common/SelectBox/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { IcArrowDown, IcArrowUp } from '@assets/icons';
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { StSelectBoxHeader, StSelectBoxWrapper, StSelectBoxItem, StSelectBoxTail } from './style';
+
+type SelectItem = {
+  id: number;
+  content: string;
+};
+
+type SelectBoxProps = {
+  items: SelectItem[];
+  selectedItemID: SelectItem['id'];
+  onItemClick: (itemID: SelectItem['id']) => void;
+};
+
+function SelectBox(props: SelectBoxProps) {
+  const { items, selectedItemID, onItemClick } = props;
+  const [isBoxOpened, setIsBoxOpened] = useState(false);
+
+  const closeSelectBox = () => {
+    setIsBoxOpened(false);
+  };
+
+  const openSelectBox = (e: React.MouseEvent) => {
+    e.stopPropagation();
+    setIsBoxOpened(true);
+  };
+
+  useEffect(() => {
+    window.addEventListener('click', closeSelectBox);
+    return () => window.removeEventListener('click', closeSelectBox);
+  }, []);
+
+  return (
+    <StSelectBoxWrapper>
+      <StSelectBoxHeader>
+        <div>{items.find((item) => item.id === selectedItemID)?.content}</div>
+        {isBoxOpened ? (
+          <IcArrowUp onClick={closeSelectBox} />
```

Comment:
```suggestion
          <IcArrowUp onClick={() => setIsBoxOpened(false)} />
```
closeSelectBox 한 줄짜리던데 요렇게 바로 쓰면 어떨까요?

Reply:
사실, `closeSelectBox`는 .. `openSelectBox`를 외롭게 하지 않기 위해 만들어준 운명의 짝꿍입니다 ..
는 사실 위의 useEffect에 쓰이는 함수라 여기서도 써주었어요~!! 그래서 쓴 김에 같이 썼습니다!!

Reply:
아하아하 제가 그 부분은 못 봤네용 좋습니다 !!!!

Reply:
외롭지 않도록 짝꿍을 만들어주다니........ 선